src/Patches/ItemRandomizer.cs

```diff
@@ -385,13 +356,8 @@ public class ItemRandomizer {
             SphereZero = FullInventory;
 
             if (SaveFile.GetInt("randomizer entrance rando enabled") == 1) {
-                List<string> sphere_zero_list = GetERSphereOne();
                 SphereZero.Clear();
-                foreach (string sphere_zero_item in sphere_zero_list) {
-                    if (!SphereZero.ContainsKey(sphere_zero_item)) {
-                        SphereZero.Add(sphere_zero_item, 1);
-                    }
-                }
+                SphereZero = AddDictToDict(SphereZero, GetERSphereOne());
```

Comment:
Technically you should just be able to call the method here without needing to do `SphereZero =`. Any changes to the dictionary inside the method are persisted outside of the method because it's passed by reference
```suggestion
                AddDictToDict(SphereZero, GetERSphereOne());
```

Reply:
Meaning you don't really have to return the dictionary from any of those methods, since it's a bit redundant